usb/cherryusb/host/host_cdc_msc_hid/usb_config.h

```diff
@@ -0,0 +1,295 @@
+/*
```

Comment:
It seems that this `usb_config.h` is identical to the other `usb_config.h` - perhaps it makes sense to add just one copy of the file to the repo (in `usb/cherryusb/` ?), and have both examples share it? :thinking: 
(i.e. like what is done with https://github.com/raspberrypi/pico-examples/blob/develop/freertos/FreeRTOSConfig_examples_common.h and https://github.com/raspberrypi/pico-examples/blob/develop/pico_w/wifi/lwipopts_examples_common.h and https://github.com/raspberrypi/pico-examples/blob/develop/pico_w/wifi/mbedtls_config_examples_common.h )

Reply:
This is an optional config for users to modify.

Reply:
Like enable log, enable bufsize for performance, enable more class instances and so on.

Reply:
Still, _perhaps_ it makes sense to have a per-example `usb_config.h` (where the users can add their optional customisation) which then does `#include "usb_config_examples_common.h"` ? (just to reduce the amount of duplication)

Reply:
> Still, _perhaps_ it makes sense to have a per-example `usb_config.h` (where the users can add their optional customisation) which then does `#include "usb_config_examples_common.h"` ? (just to reduce the amount of duplication)

I know what you mean, yeah, reduce the duplication, also freertos. I will modify.

Reply:
Have done. And i see pico_w demo has `FreeRTOSConfig_examples_common.h`, so i move this into config dir.

Reply:
> Have done.

Thanks :+1:  But I'm still not sure if it makes sense to have both `freertos/FreeRTOSConfig_examples_common.h` and  `usb/cherryusb/config/FreeRTOSConfig_examples_common.h`, which appear to be identical copies of each other?